R/qExportWig.R

```diff
@@ -22,47 +22,207 @@
 # absIsizeMax : maximum absolute insert size (TLEN <= absIsizeMax)
 # useRead     : for paired-end data, what read to use
 # pairedAsSingle : for paired-end data, treat as single read data (do not calculate fragment mid-points)
+#' QuasR wig file export
+#' 
+#' Create a fixed-step wig file from the alignments in the genomic bam files 
+#' of the \sQuote{QuasR} project.
+#' 
+#' \code{qExportWig()} uses the genome bam files in \code{proj} as input
+#' to create wig or bigWig files with the number of alignments (pairs)
+#' per window of \code{binsize} nucleotides. By default
+#' (\code{collapseBySample=TRUE}), one file per unique sample will be
+#' created. If \code{collapseBySample=FALSE}, one file per genomic bam
+#' file will be created. See \url{http://genome.ucsc.edu/goldenPath/help/wiggle.html} 
+#' for the definition of the wig format, and 
+#' \url{http://genome.ucsc.edu/goldenPath/help/bigWig.html} for the definition 
+#' of the bigWig format.
+#' 
+#' The genome is tiled with sequential windows of length \code{binsize},
+#' and alignments in the bam file are assigned to these windows: Single
+#' read alignments are assigned according to their 5'-end coordinate
+#' shifted by \code{shift} towards the 3'-end (assuming that the 5'-end
+#' is the leftmost coordinate for plus-strand alignments, and the rightmost
+#' coordinate for minus-strand alignments). Paired-end alignments are
+#' assigned according to the base in the middle between the leftmost and
+#' rightmost coordinates of the aligned pair of reads. Each pair of reads
```

Comment:
This is not completely correct anymore. It should read: "... rightmost coordinates of the aligned pair of reads, unless \code{pairedAsSingle = TRUE} is used."

Reply:
Fixed.